src/gluonts/model/deepar/_schema.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import Any, Dict, Optional, Union, Generic, TypeVar, Type
+
+import numpy as np
+import pandas as pd
+from dataclasses import dataclass
```

Comment:
Minor: We want to keep the imports in three blocks: 1st party (provided by python), 3rd party (our dependencies that we installed, and then imports from `gluonts`.

Reply:
In this case this would look like:

```py
from dataclasses import dataclass
from typing import Any, Dict, Optional, Union, Generic, TypeVar, Type

import numpy as np
import pandas as pd
from pandas.tseries.frequencies import to_offset

from gluonts.exceptions import GluonTSDataError
```